eo-runtime/src/main/eo/org/eolang/bytes.eo

```diff
@@ -7,6 +7,13 @@
 +version 0.0.0
 +spdx SPDX-FileCopyrightText Copyright (c) 2016-2025 Objectionary.com
 +spdx SPDX-License-Identifier: MIT
++unlint redundant-object:23
```

Comment:
@h1alexbel it seems we can remove some of these `+unlint` meta

Reply:
@h1alexbel you may not touch such self referencing objects, there are a lot of other cases